tla-bmcmt/src/main/scala/at/forsyte/apalache/tla/bmcmt/rules/ChooseRule.scala

```diff
@@ -75,74 +73,5 @@ class ChooseRule(rewriter: SymbStateRewriter) extends RewritingRule {
 
   // This translation is sound only in the happy scenario, that is, when CHOOSE is defined on its argument.
   // It should be used only after resolving the issue #95: https://github.com/informalsystems/apalache/issues/95
-  private def happyChoose(
-      state: SymbState,
-      varName: String,
-      set: TlaEx,
-      pred: TlaEx): SymbState = {
-    // rewrite the bounding set
-    val nextState = rewriter.rewriteUntilDone(state.setRex(set))
-    val setCell = nextState.asCell
-    val isFinSet = PartialFunction.cond(setCell.cellType) { case FinSetT(_) => true }
-    if (isFinSet) {
-      happyChooseFromFinite(nextState, setCell, varName, pred)
-    } else {
-      happyChooseFromOther(nextState, setCell, varName, pred)
-    }
-  }
-
```

Comment:
Is this something we want to keep around, @konnov? I think we could remove the `private` qualifier if so.

Reply:
Yes, we should remove this unused definition.

Reply:
and the one below too.